documentation/sphinx/source/release-notes/release-notes-620.rst

```diff
@@ -4,6 +4,14 @@
 Release Notes
 #############
 
+6.2.27
+======
+
+Fixes
```

Comment:
We no longer add the header in release notes. (r.f. 6.2.27 release note). 

Would you mind removing it? Thanks!